activerecord/test/cases/migration/index_test.rb

```diff
@@ -256,6 +256,22 @@ def test_add_partial_index
           connection.remove_index("testings", "last_name")
           assert_not connection.index_exists?("testings", "last_name")
         end
+
+        def test_add_index_with_included_column
+          connection.add_index("testings", "last_name", include: :foo)
+          assert connection.index_exists?("testings", "last_name", include: :foo)
```

Comment:
`index_exists?` should be extended to also add a check for `include`. 
If you rewrite the first line with `connection.add_index("testings", "last_name")`, the second line will still pass.

Reply:
Nice catch! I updated `index_exists?` with an additional check to cover the `include` option.